components/Header/Header.tsx

```diff
@@ -9,13 +9,13 @@ import Tooltip from "../ui/Tooltip";
 
 const Header = () => {
   const { push } = useRouter();
-  const { address, connectWallet } = useConnectWallet();
+  const { authenticated, ready, login } = useConnectWallet();
```

Comment:
please import directly from `usePrivy`
```suggestion
  const { authenticated, ready, login } = usePrivy();
```